src/main/java/ru/skypro/homework/entity/Comments.java

```diff
@@ -0,0 +1,67 @@
+package ru.skypro.homework.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Getter
+@Table(name = "comments")
+public class Comments {
```

Comment:
```suggestion
public class Comment {
```
названия классов должны быть в единственном числе